string pointcloud_filepath
sensor_msgs/PointCloud2 point_cloud
geometry_msgs/TransformStamped point_cloud_to_world_stamped